src/Chainweb/Pact/RestAPI/Server.hs

```diff
@@ -349,7 +349,7 @@ localHandler logger pact preflight cmd = do
     case r of
       Left err ->
         throwError $ err400 { errBody = "Execution failed: " <> BSL8.pack (show err) }
-      (Right !r') -> return r'
+      (Right !r') -> error "branch on metadata error formatting or command"
```

Comment:
We should never call `error`; I assume this is just a temporary hole to be filled with real code?

Reply:
This was a todo to stub out how we were going to handle metadata validations. It's fixed now